### PR TITLE
Adds wait loop to entrypoint when --multus-conf-file=auto 

### PR DESCRIPTION
**Pull request by dougbtv**
Waits for a file to be present in the CNI configuration directory when using the `--multus-conf-file=auto `

### CODE REVIEW

File: images/entrypoint.sh [hunk 1]
```diff
@@ -151,27 +151,40 @@ fi
...
-		)
-    echo $CONF > $CNI_CONF_DIR/00-multus.conf
-  fi
+  found_master=false
+  tries=0
+  while [ $found_master = false ]; do
+    MASTER_PLUGIN="$(ls $CNI_CONF_DIR | grep -E '\.conf(list)?$' | head -1)"
+    if [ "$MASTER_PLUGIN" == "" ]; then
+      if [ $tries -lt 600 ]; then
+        echo "Attemping to find master plugin configuration, attempt $tries"
```
Comment (dougbtv) [inline images/entrypoint.sh]: Script is just bombing out after this....

**Comment by dougbtv** [comment]
This might not even be necessary -- you can just let the pod crash loop and it'll start when it finds the file. This allows to use the Kubernetes back-off and not write any more logic here. 

Current plan: Close this issue.

**Comment by dougbtv** [comment]
Reopened as this does actually cause people some consternation during a deployment overall with more pods crashlooping, when it's totally OK for it to just wait. New plan: Re-test, rebase, review and merge.

**Comment by dougbtv** [comment]
This PR is ready to rumble, OK to merge at any time (or, pending review)

File: images/entrypoint.sh [hunk 2]
```diff
@@ -157,32 +157,47 @@ fi
...
-    exit 1;
-  elif [ "$MASTER_PLUGIN" == "00-multus.conf" ]; then
-    echo "Warning: Multus is already configured: auto configuration skipped."
-  else
-    ISOLATION_STRING=""
-    if [ "$MULTUS_NAMESPACE_ISOLATION" == true ]; then
-      ISOLATION_STRING="\"namespaceIsolation\": true,"
+  found_master=false
+  tries=0
+  while [ $found_master = false ]; do
```
[the remark this answers is not in the document]
Comment (dcbw) [reply images/entrypoint.sh]: I think you can do !$found_master too? Maybe :)

**Pull request merged**